docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-product-option-feature-integration.md

```diff
@@ -0,0 +1,465 @@
+
+class ProductOptionStorageDependencyProvider extends SprykerProductOptionStorageDependencyProvider
+{
+    /**
+     * @return \Spryker\Zed\ProductOptionStorageExtension\Dependency\Plugin\ProductOptionCollectionFilterPluginInterface[]
+     */
+    protected function getProductOptionCollectionFilterPlugins(): array
+    {
+        return [
+            new MerchantProductOptionCollectionFilterPlugin(),
+        ];
+    }
+}
+```
+
+**src/Pyz/Zed/Publisher/PublisherDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Zed\Publisher;
+
+use Spryker\Zed\MerchantProductOptionStorage\Communication\Plugin\Publisher\MerchantProductOption\MerchantProductOptionGroupWritePublisherPlugin;
+use Spryker\Zed\Publisher\PublisherDependencyProvider as SprykerPublisherDependencyProvider;
+
+class PublisherDependencyProvider extends SprykerPublisherDependencyProvider
+{
+    /**
+     * @return array
+     */
+    protected function getPublisherPlugins(): array
+    {
+        return array_merge(
+            // ...
+            $this->getMerchantProductOptionStoragePlugins()
+        );
+    }
+    
+    /**
+     * @return \Spryker\Zed\PublisherExtension\Dependency\Plugin\PublisherPluginInterface[]
+     */
+    protected function getMerchantProductOptionStoragePlugins(): array
+    {
+        return [
+            new MerchantProductOptionGroupWritePublisherPlugin(),
+        ];
+    }
+}
+```
+
+{% info_block warningBox "Verification" %}
+
+Make sure merchants can create product option groups and values in the merchant portal.
+Make sure that merchant product option information is shown on product details page, when it is approved and active.
+Make sure that merchant product option information is displayed in the cart, checkout and in the user account.
+Make sure that merchant product options are part of a marketplace/merchant order and all totals are calculated correctly.
+
+{% endinfo_block %}
+
+
+### 5) Add Yves translations
+
+Append glossary according to your configuration:
+
+**src/data/import/glossary.csv**
+
+```yaml
+checkout.item.option.pre.condition.validation.error.exists,"Product option of %name% is not available anymore.",en_US
+checkout.item.option.pre.condition.validation.error.exists,"Produktoption von %name% ist nicht mehr verfügbar.",de_DE
+```
+
+Import data:
+
+```bash
+console data:import glossary
+```
+
+{% info_block warningBox "Verification" %}
+
+Make sure that the configured data is added to the `spy_glossary` table in the database.
+
+{% endinfo_block %}
+
+
+### 6) Import data
+
+Prepare your data according to your requirements using the demo data:
+
+<details><summary>data/import/common/common/marketplace/merchant_product_option_group.csv</summary>
+
+```csv
+product_option_group_key,merchant_reference,approval_status,merchant_sku
+insurance,MER000001,approved,spr-425453
+```
+
+</details>
+
+#### Adjust the following files:
+
+<details><summary>data/import/common/AT/product_option_price.csv</summary>
```

Comment:
This is a part of `Product Option Feature Integration` guide, should not be present in `Marketplace Product Option Feature Integration` guide.